server/sql/dev/create-tables-dev.sql

```diff
@@ -1,18 +1,47 @@
 \c socobo_dev;
 
+--
+--  SOCOBO USER
+--
+
+CREATE TABLE Socobo_User_Role (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR NOT NULL UNIQUE,
+  created NUMERIC NOT NULL,
+  lastModified NUMERIC NOT NULL CHECK (lastModified >= created)
+);
+
+CREATE TABLE Socobo_User_Provider (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR NOT NULL UNIQUE,
+  created NUMERIC NOT NULL,
+  lastModified NUMERIC NOT NULL CHECK (lastModified >= created)
+);
+
+CREATE TABLE Socobo_User_Image (
```

Comment:
> How do we know the id of the user image? The user isn't registered at this moment and never has added a image.

a new user has always the default user image (=== profile pic) which is stored inside our db. if the user like he could change this and upload a new image with a new id and this id would update socobo_user. 

> If you want to refer to a default image in this case I would prefer to define this url as a constant in the config because if we want to change the image we would need to change it in the db what would also change the id.

we could change the default with a simple sql update. this  doesn't update the id.

> So we would also need to do a request to figure out the current id of the default image.

default id is stored inside the Config class